src/main/java/pl/stormit/tinyurl/domain/model/Url.java

```diff
@@ -5,23 +5,16 @@
 
 import javax.persistence.*;
 import java.time.LocalDate;
+import java.util.UUID;
 
 @Entity
 @Table(name = "urls")
 @ToString
```

Comment:
Maybe use Lombok annotation without getter and setter method in class

Reply:
I don't think it's needed. In all constructors I must add id generator:

public Url() {
        this.id = UUID.randomUUID();
    }

Reply:
OK now I understand what do you mean :)